### PR TITLE
Table julia

### PR DESCRIPTION
**Pull request by Vladomira**
Please check

### CODE REVIEW

File: src/components/Table/TableItem.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    <>
+      {
+        // transactions.length > 0 ?
+        transactions.map(
+          ({ _id, createdAt, month, type, category, comment, sum, total }) => {
+            return (
+              // <tbody className="table__body">
+              <tr key={_id} className="table__data-row">
+                <td className="table__item">{month}</td>
+                <td className="table__item">{type === true ? '+' : '-'}</td>
```
Comment (Prykhodko-Pasha) [inline src/components/Table/TableItem.js]: Why don't you write just "type ? + : -"

File: src/components/Table/TableItem.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                <td className="table__item">{category}</td>
+                <td className="table__item">{comment}</td>
+                <td
+                  className={
+                    type === true
```
Comment (Prykhodko-Pasha) [inline src/components/Table/TableItem.js]: The same is here

File: src/redux/functionServise.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export function getTotalBalance(payload) {
+  //   console.log('payload', payload);
+  const byType = payload.map(el => (el.type === true ? el.sum : null));
```
Comment (Prykhodko-Pasha) [inline src/redux/functionServise.js]: The same is here

File: src/redux/table/table-actions.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { createAction } from '@reduxjs/toolkit';
+
+const fetchTransactions = createAction('transactions/fetch');
+
+const getBalance = createAction('balance/getBalance');
+
+const actions = { fetchTransactions, getBalance };
+
+export default actions;
```
Comment (Prykhodko-Pasha) [inline src/redux/table/table-actions.js]: Please rename this folder and files inside to "transactionsTable" or something like this

File: src/redux/functionServise.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  //     if (previousValue === 'number' && current === 'number')
+  //       console.log('previousValue', previousValue, current);
+  //     // return previousValue + current;
+  //     return current;
+  //   });
+  //   console.log('sum', sum);
+
+  //   return sum;
+  return 320;
+}
```
Comment (Prykhodko-Pasha) [inline src/redux/functionServise.js:16]: I think this function is responsible for getting user balance, so rename it according to it's purpose

File: src/views/TableView.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  );
+}
+
+//    {/* {table.length > 0 ? (
+//         <>
+//           <Table />
+//         </>
+//       ) : (
+//         <p className="section__title">Loading...</p>
+//       )} */}
```
Comment (Prykhodko-Pasha) [inline src/views/TableView.js]: Please rename this component

Comment (Vladomira) [reply src/views/TableView.js]: done

Comment (Vladomira) [reply src/views/TableView.js]: done

File: src/redux/functionServise.js (hunk 5 above)
Comment (Vladomira) [reply src/redux/functionServise.js:16]: renamed to"userBalace"

File: src/redux/table/table-actions.js (hunk 4 above)
Comment (Vladomira) [reply src/redux/table/table-actions.js]: renamed

File: src/components/Table/TableItem.js (hunk 1 above)
Comment (Vladomira) [reply src/components/Table/TableItem.js]: changed, thanks

File: src/components/Table/TableItem.js (hunk 2 above)
Comment (Vladomira) [reply src/components/Table/TableItem.js]: thanks

File: src/redux/functionServise.js (hunk 3 above)
Comment (Vladomira) [reply src/redux/functionServise.js]: resolved

**Review by od-frontender — COMMENTED** [review]
good

**Review by od-frontender — APPROVED** [review]

**Review by od-frontender — APPROVED** [review]

**Review by Prykhodko-Pasha — APPROVED** [review]

**Pull request merged**